### PR TITLE
I made Chutulu bounce around a black screen with red blinking words…

### PR DESCRIPTION
**Pull request by smitzter**
…that threaten the viewer bc why not... ok the assignment was to animate the page somehow.
#4 fixed issue by adding animation.

I also cleaned up the code a lot but it still does not work.

### CODE REVIEW

File: js/app.js [hunk 1]
```diff
@@ -1,239 +1,174 @@
...
-  this.shown = 0
 }
 
-function generateRandomProduct() {
-  var index = Math.floor(Math.random() * productNames.length);
-  return productNames[index];
+//put three img into placing///
+function renderProductImage (products, imgAll) {
+  var x = document.getElementById('id'); document.createAttribute('src');
+  x.value = 'filepath';
```
Comment (sueanyv) [inline js/app.js:92]: linter warning says the file path is null.

File: js/app.js [hunk 2]
```diff
@@ -1,239 +1,174 @@
...
+new MakeImage('unicorn', 'img/unicorn.jpg', 0, 0),
+new MakeImage('usb', 'img/usb.jpg', 0, 0),
+new MakeImage('waterCan', 'img/waterCan.jpg', 0, 0),
+new MakeImage('wineGlass', 'img/wineGlass.jpg', 0, 0);
+
+
+///functions//////////////////////
+
+//Limit Attempts///
+if (attempts === maxAttempts) {
```
Comment (wrathofdon) [inline js/app.js:63]: I would make it >= instead of === just in case.

File: js/app.js [hunk 3]
```diff
@@ -1,239 +1,174 @@
...
-  // for (var p = 0; p < 3; i++) {
-  //   productNames ;
-  // }
-  renderProductImage(productImage1);
-  renderProductImage(productImage2);
-  renderProductImage(productImage3);
-  // updateAnswer();
-  // updateAttempts();
+///calls////
+renderProductImage();
```
Comment (wrathofdon) [inline js/app.js:22]: I'm not sure why you need so many calls right here.  Also, I'm not clear on what a lot of these functions actually do.

File: js/app.js [hunk 4]
```diff
@@ -1,239 +1,174 @@
...
-  if (attempts === maxAttempts) {
-    chart();
+function imgPath() {
+  var ip = MakeImage.setAttribute('src');
+  ip.value = product.path;
+  img.setAttributeNode(ip);
+}
+//render with math random///
+//check for dups///
+function render() {
```
Comment (wrathofdon) [inline js/app.js:76]: This is where you should start incrementing the number of times shown.

File: js/app.js [hunk 5]
```diff
@@ -1,239 +1,174 @@
...
+  var randomImg = function() {
+    img1 = Math.floor(Math.random() * imgPath.length);
+    img2 = Math.floor(Math.random() * imgPath.length);
+    img3 = Math.floor(Math.random() * imgPath.length);
+    newImg = [];
+    pushImg.push(img1, img2, img3);
+  };
+  while ( event.target.id === img1.name || event.target.id === img2.name || event.target.id === img3.name) {
```
Comment (wrathofdon) [inline js/app.js:84]: This doesn't need to be a while loop.

**Review by wrathofdon — COMMENTED** [review]
Added some suggestion on how to fix code.

File: js/app.js (hunk 1 above)
Comment (traceyr) [inline js/app.js:92]: Your error:
Uncaught TypeError: Cannot set property 'value' of null
    at renderProductImage (app.js:92)
    at app.js:22

**Pull request merged**